README.md

```diff
@@ -582,6 +587,8 @@ For a real-life example, you can check my personal dots:
 - `lock` ➡️ `pin`
 - `disable=true` ➡️ `enabled = false`
 - `tag='*'` ➡️ `version="*"`
+- `wants` ⚠️ **Not supported**
```

Comment:
perhaps write that you can accomplish this with `dependencies`

Reply:
agree, `wants` and `after` are simply not needed with `lazy`. For lua plugins even `dependencies` aren't needed.
So it's not that it's not supported, it's simply not needed.
Now it sounds like this is a limitation of lazy, while deps and especially auto-loading of lua deps are a big improvement over packer

Reply:
perhaps this should be written somewhere
the thing that you basically add dependencies anywhere because of the "automatic `module` lazyloading"
for me this was clear but ig for other people who don't know much about how this things work internally it wasn't

Reply:
I think `requires` is direct translation to `dependencies` while `wants` is undocumented options that some plugins suggest in their configs.

It's not needed for `lazy` spec. I just delete all `want` and change `requires` to `dependencies`. So far so good for my config

Reply:
I just include it so people will check those keys in their spec. Maybe I should change to **not required**?

Reply:
even most of the requires are not needed.

Reply:
Maybe:
**not needed** for most cases, otherwise use `dependencies`.